.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Pull Request Desription
+
+## Check if you branch history is PR compatible
+
+- Your branch need to be up to date with origin/main
+- Update your git repository `git fetch origin` if origin is this remote
+- Check format style with the script provided in `scripts/source-format-all.sh`
+- You must use clang-format style (github CI version is 11)
+_These checks are enforced by github workflow actions_
+_Please refer to the corresponding log in case of failure_
+
+## UPDATE the form below to describe your PR
+
+Please check if the PR fulfills these requirements
+
+- [ ] The commit message follows our guidelines
```

Comment:
Do we have guidelines for the commits ?

Reply:
Yes, the guidelines for commit is in CONTRIBUTING.md file. (line 141)

Reply:
Please link to this file in the PR template

Reply:
Can we refer to other md file in md file ?